products/bigquery/api.yaml

```diff
@@ -383,6 +383,613 @@ objects:
               A-Z), numbers (0-9), or underscores (_). The maximum length
               is 1,024 characters.
             required: true
+  - !ruby/object:Api::Resource
+    name: 'Job'
+    kind: 'bigquery#job'
+    base_url: projects/{{project}}/jobs
+    delete_url: projects/{{project}}/jobs/{{job_id}}/cancel
+    delete_verb: :POST
+    self_link: projects/{{project}}/jobs/{{job_id}}
+    input: true
+    description: |
+      Jobs are actions that BigQuery runs on your behalf to load data, export data, query data, or copy data.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'etag'
+        description: |
+          A hash of the resource.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'id'
```

Comment:
@emilymye  I didn't know that was a thing, I'll have to look into how that works?  I tried it, but got an error, so I must not understand it completely.  But, I'm wondering, would this be conflicting with when I flatten `jobReference`?  Because `jobReference` has a `jobId` that can be set.  I'm assuming they'd end up with the same value, but I'd have to double-check that.